omero/developers/Web/Deployment.txt

```diff
@@ -80,7 +81,6 @@ listening on 127.0.0.1:4080 that will autoreload on source change:
 ::
 
     $ nginx -c $PWD/nginx-development.conf
-    $ bin/omero config set omero.web.application_server 'wsgi-tcp'
     $ bin/omero config set omero.web.application_server.max_requests 1
     $ bin/omero web start  --wsgi-args ' --reload'
```

Comment:
as changed in https://github.com/openmicroscopy/openmicroscopy/pull/4352 lets move it to omero properties